src/lib/components/circularprogressbar/CircularProgressBar.component.style.js

```diff
@@ -0,0 +1,59 @@
+import styled from "styled-components";
+import { mergeTheme } from "../../utils";
+import * as defaultTheme from "../../style/theme";
+
+export const CircularProgressBarContainer = styled.div`
+  margin: 0;
+  padding: 0;
+  color: ${props => mergeTheme(props.theme, defaultTheme).text};
+  svg 
+  {
+    position: relative;
+    width: 150px;
+    height: 150px;
+    z-index: 1000;
+  }
+  svg circle 
+  {
+    width: 100%;
+    height: 100%;
+    fill: none;
+    stroke: ${props => mergeTheme(props.theme, defaultTheme).text};
+    stroke-width: 10;
```

Comment:
`stroke-width` should be be defined as a prop (ex: circleThickness). You can put a default value (ex: 10)